benchmarks/benchmarks/optimize_linprog.py

```diff
@@ -243,3 +247,32 @@ def track_netlib_rr(self, meth, prob):
             return float(self.error1)
         else:
             return float(self.error2)
+
+
+class Netlib_presolve(Benchmark):
+    params = [
+        presolve_methods,
+        presolve_problems
+    ]
+    param_names = ['method', 'problems']
+
+    def setup(self, meth, prob):
+
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        data = np.load(dir_path + "/linprog_benchmark_files/" + prob + ".npz",
```

Comment:
Is this path concatenation portable to Windows?  might need an `os.path.join` somewhere in there?

Reply:
Sorry, I'm forgetting... @Kai-Striega are you asking me to change this, too, or do you mean that it is right here because we fixed it previously in the `Netlib` and `Netlib_RR` benchmarks ?
I tested this on Windows already, actually, so I think it is fine on Windows.

Reply:
The problem is that Windows uses forward slashes to indicate options from the command line, rather than the `-` as is common for nix base systems.

>  are you asking me to change this, too, or do you mean that it is right here because we fixed it previously

This should be fixed. 

> I tested this on Windows already, actually, so I think it is fine on Windows.

I'm surprised that it works in Windows. I've just tried running it and the forward slashes work from Python but not from the CLI. Doing a bit of reading it looks like Python tries to "guess" the correct slash automatically, but it's still best to use os.path

Reply:
Oops, I see. Thanks. I see that I did fix this before for `Netlib` but didn't change `Netlib_RR`, from which I created this class. (I misunderstood because I remembered we had fixed it before, so I couldn't figure out how this was wrong.)